wiki/misc/worldtypes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Adding a Custom World Preset
+---
+
+# Adding custom world presets
+If your mod completely overhauls Minecraft's worldgen, you'll want to create a custom worldtype so you give people the option to use your worldgen or MC's vanilla worldgen.
+Now, what is a custom worldtype you may ask? World types define world generation options for Minecraft worlds. They let you change the make-up of your Minecraft worlds by configuring how world generation behaves.
+
+</br>
+</br>
+
+Here is where you specify which dimensions your world preset will apply to. It does require a `minecraft:overworld` dimension or your world preset will not work.
+Valid dimensions are `minecraft:overworld`, `minecraft:nether` and `minecraft:the_end`.
+```json
+"dimensions": {
+  "minecraft:overworld": {
+    "type": "minecraft:overworld"
```

Comment:
The type here isnt explained. Its the actual world type. In addition, the name of the object is the dimension name, and besides requiring `minecraft:overworld`, nothing else is. Make this distinction clearer.